app/views/grade_entry_forms/_list.html.erb

```diff
@@ -14,41 +14,7 @@
       </thead>
       <tbody>
         <% grade_entry_forms.each do |grade_entry_form| %>
-          <% gef_text = "#{h(grade_entry_form.short_identifier)}: #{h(grade_entry_form.description)}" %>
```

Comment:
Hmm, is this partial even used anywhere any more? If not, we could delete it altogether.